doc/source/whatsnew/v0.18.2.txt

```diff
@@ -524,3 +524,5 @@ Bug Fixes
 
 
 - Bug in ``Categorical.remove_unused_categories()`` changes ``.codes`` dtype to platform int (:issue:`13261`)
+
+- Bug in ``tslib.dates_normalized`` returns False in case of normalized date_range for local timezone  ``Asia/Kolkata`` (:issue:`13459`)
```

Comment:
From user's point of view, it should describe the change related to ``DatetimeIndex.is_normalized``. Also, it affects to all tz (not only to Kolkata).